.eslintrc.js

```diff
@@ -1,8 +1,8 @@
 // prettier-ignore
 module.exports = {
   parser: 'babel-eslint',
-  plugins: ['flowtype', 'react'],
-  extends: ['uber-jsx', 'uber-es2015', 'prettier', 'prettier/react', 'prettier/flowtype', 'plugin:import/errors', 'plugin:flowtype/recommended'],
+  plugins: ['react', '@typescript-eslint'],
```

Comment:
I had bad experience with using the eslint typescript integration (many corner cases and bad perf). Switched to `tsc --noEmit`.

Reply:
math.gl is still using this plugin. I haven't observed any problem so far.